WalletWasabi/Services/UpdateManager.cs

```diff
@@ -18,6 +19,7 @@ public class UpdateManager : IDisposable
 	private string InstallerPath { get; set; } = "";
 	private const byte MaxTries = 3;
 	private const string ReleaseURL = "https://api.github.com/repos/zkSNACKs/WalletWasabi/releases/latest";
+	private CancellationToken _cancellationToken;
```

Comment:
please use a property instead - code convention.

Reply:
isn't it nullable?

Reply:
No, CancellationToken's default value is `CancellationToken.None`.